### DISCUSSION TITLE
Unable to input password in tencent wegame app on Windows 10

### DISCUSSION THREAD

**Discussion by JerryMoon** [rustdesk/rustdesk#8910] [General]
### Bug Description

Unable to input password in tencent wegame app on Windows 10.
And it can input normal text, eg. account text.
And after clicking the mouse, the password input box can get focus.

### How to Reproduce

1、android or windows10 -> windows10
2、input password

### Expected Behavior

can input password

### Operating system(s) on local side and remote side

android/windows10 -> windows10

### RustDesk Version(s) on local side and remote side

1.2.4->

### Screenshots

<img width="342" alt="1111" src="https://github.com/user-attachments/assets/833c6cbb-5751-4f8b-880b-043fccafe71a">

**Comment by rustdesk** [comment]
I do not think this is not what we can control, or it requires special API call, anyhow we won't spend time on this corner case. But we welcome PR.
